standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestAddPartitionsFromPartSpec.java

```diff
@@ -265,18 +265,17 @@ public void testAddPartitionSpecUpperCaseDBAndTableNameInOnePart() throws Except
 
     String rootPath = tableLocation + "/addpartspectest/";
     PartitionSpecProxy partitionSpec = buildPartitionSpec(DB_NAME, tableName, rootPath, partitions);
-    client.add_partitions_pspec(partitionSpec);
+    try {
+      client.add_partitions_pspec(partitionSpec);
+      Assert.fail("MetaException should have been thrown.");
+    } catch (MetaException e) {
+      // Expected exception
+      System.out.println(e);
```

Comment:
Can you log this instead of using System.out?

Reply:
Replace it with an exception message check.